src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -133,6 +142,42 @@ export default class BraintreeHostedForm {
         }
     }
 
+    async tokenizeWith3DSRegulationCheck(billingAddress: Address, merchantAccountId: string): Promise<Pick<BraintreeHostedFieldsTokenizePayload, 'authenticationInsight'> & NonceInstrument> {
```

Comment:
BraintreeHostedFieldsTokenizePayload has the type `details` but this method doesn't return any `details`, would the pick be necessary in this case?

Reply:
I used Pick to get 'authenticationInsight' type from BraintreeHostedFieldsTokenizePayload interface. But, I think, we can create another interface to handle this case

Reply:
Fixed. Thanks you for your comment 🙏